packages/amplify-provider-awscloudformation/src/root-stack-builder/root-stack-transform.ts

```diff
@@ -1,4 +1,4 @@
-import { AmplifyRootStackTemplate } from './types';
+import { AmplifyRootStackTemplate } from '@aws-amplify/cli-extensibility-helper';
```

Comment:
why are we using the root-stack template in two places? in transform as well as the builder?